### PR TITLE
Feat add path setting

### PR DESCRIPTION
**Pull request by yanguoyu**
1. Set ckb node data path

https://user-images.githubusercontent.com/12881040/176328888-cffd9f15-211d-4458-a162-e5a8677725b9.mov

2. Set indexer data path

https://user-images.githubusercontent.com/12881040/176329102-9613f43e-71db-48f7-b240-6f58a0d01480.mov

3. Open data path

https://user-images.githubusercontent.com/12881040/177074313-d36ecc39-e1bf-40ce-a1b2-c96a0c896d90.mov

### CODE REVIEW

File: packages/neuron-ui/src/services/remote/app.ts [hunk 1]
```diff
@@ -9,6 +17,12 @@ export const requestOpenInExplorer = remoteApi<Controller.RequestOpenInExplorerP
 export const handleViewError = remoteApi<string>('handle-view-error')
 export const showSettings = remoteApi<Controller.ShowSettingsParams>('show-settings')
 export const setLocale = remoteApi<typeof LOCALES[number]>('set-locale')
+export const getCkbNodeDataPath = remoteApi<undefined, string>('get-ckb-node-data-path')
```
Comment (Keith-CY) [inline packages/neuron-ui/src/services/remote/app.ts:20]: `undefined` could be replaced by `void`

File: packages/neuron-ui/src/components/DataSetting/hooks.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  useDidMount(() => {
+    getPath(undefined).then(res => {
+      if (isSuccessResponse(res)) {
+        setPrevPath(res.result!)
+      }
+    })
+  })
+  const onSetting = useCallback(() => {
+    invokeShowOpenDialog({
+      buttonLabel: t('settings.data.set', { lng: navigator.language }),
```
[the remark this answers is not in the document]
Comment (yanguoyu) [reply packages/neuron-ui/src/components/DataSetting/hooks.ts:32]: Maybe system language is different from user-select in Neuron. This button label is shown with system dialog.

[the remark this answers is not in the document]
Comment (Keith-CY) [reply packages/neuron-ui/src/components/DataSetting/hooks.ts:32]: Make sense

**Review by Keith-CY — APPROVED** [review]

**Review by Cedar67 — APPROVED** [review]
Test Results: Verified.
Test case & results: 
https://github.com/Magickbase/neuron-public-issues/issues/13#issuecomment-1173584917

**Pull request merged**